test/lint.ts

```diff
@@ -25,17 +26,15 @@ const dirname = fileURLToPath(new URL('.', import.meta.url));
  * @param {string[]} files The files to test
  * @returns {DataType?}
  */
-const loadAndCheckFiles = async (
-  ...files: string[]
-): Promise<DataType | undefined> => {
+const loadAndCheckFiles = async (...files: string[]): Promise<DataType> => {
```

Comment:
This could still return `undefined` on line 43, but I'm now realizing that this is a bug, and it should really be a `continue;` rather than a `return;`.  Could we fix that while we're here?